VersionControlUI/src/nodes/DiffComparisonPair.h

```diff
@@ -61,27 +63,37 @@ class VERSIONCONTROLUI_API DiffComparisonPair : public Super<Visualization::UINo
 			Model::Text* newVersionObjectPath();
 			Model::Text* oldVersionObjectPath();
 
+			Model::Text* componentType();
+
 
 		private:
 			Model::Node* oldVersionNode_;
 			Model::Node* newVersionNode_;
+
 			Model::Text* newVersionObjectPath_;
 			Model::Text* oldVersionObjectPath_;
 
+			Model::Text* componentType_;
```

Comment:
I should have mentioned this earlier: please always default initialize `{}` pointers. Do this for all members. It helps avoid stupid mistakes sometimes.